gordo_components/dataset/datasets.py

```diff
@@ -21,6 +21,10 @@
 logger = logging.getLogger(__name__)
 
 
+class InsufficientDataError(ValueError):
```

Comment:
Also remember to add it to this handy map: https://github.com/equinor/gordo-components/blob/50d5a9e96550b2c9c4eda02bcc07c6f725da7cca/gordo_components/cli/cli.py#L31

Reply:
Oooh... right.